src/SearchResults.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const SearchResult = props => {
+  return (
+    <table className="table">
+      <thead>
+        <tr>
+          <th scope="col">Id</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room Id</th>
+          <th scope="col">Check In Date</th>
+          <th scope="col">Check Out Date</th>
+          <th scope="col">Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((item, index) => (
+          <TableRow item={item} index={index} />
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+const TableRow = props => {
+  const [selected, setSelected] = useState(false);
+
+  const clickHandler = () => {
+    selected === false ? setSelected(true) : setSelected(false);
+  };
```

Comment:
Good use of a ternary! You could shorten this to 
```
const clickHandler = () => {
    setSelected(!selected);
};
```

Reply:
Thanks Ellie, makes more sense , I changed it.